cli/lib/kontena/cli/common.rb

```diff
@@ -177,7 +177,7 @@ def client(token = nil, api_url = nil)
 
         @client ||= Kontena::Client.new(
           api_url || require_current_master.url,
-          token || require_current_master.token
+          token || ENV['KONTENA_TOKEN'] || require_current_master.token
```

Comment:
AKA with this change, just setting `KONTENA_TOKEN=asdf` will actually cause the CLI to fail with HTTP 403. Probably not a bug, seems intended.